rest/taskana-rest-spring/src/main/java/pro/taskana/rest/TaskController.java

```diff
@@ -442,6 +468,22 @@ private void checkForIllegalParamCombinations(MultiValueMap<String, String> para
               + PLANNED_UNTIL
               + "\"");
     }
+
+    if (params.containsKey(WILDCARD_SEARCH_FIELDS) && !params.containsKey(WILDCARD_SEARCH_VALUE)
+        || !params.containsKey(WILDCARD_SEARCH_FIELDS)
+            && params.containsKey(WILDCARD_SEARCH_VALUE)) {
+
+      throw new IllegalArgumentException(
+          "It is prohibited to use the params "
```

Comment:
I think the message would be more understandable if you only point out that both parameters have to be present if used.